docs/dev/metrics.md

```diff
@@ -59,10 +59,10 @@ cluster_operator_payload_errors{version="4.0.3"} 10
 
 Metrics about the installation:
 
-`cluster_installer` records information about the installation process. The type is either "openshift-install", indicating that `openshift-install` was used to install the cluster (IPI) or "", indicating that an unknown process installed the cluster (UPI). When `openshift-install` creates a cluster, it will also report its version and invoker. The version is helpful for determining exactly which builds are being used to install (e.g. were they official builds or had they been modified). The invoker is "user" by default, but it may be overridden by a consuming tool (e.g. Hive, CI).
+`cluster_installer` records information about the installation process. The type is either "openshift-install", indicating that `openshift-install` was used to install the cluster (IPI) or "other", indicating that an unknown process installed the cluster (UPI). When `openshift-install` creates a cluster, it will also report its version and invoker. When an unknown process installed the cluster, the version and invoker reported will be that of the `openshift-install` invocation which created the manifests. The version is helpful for determining exactly which builds are being used to install (e.g. were they official builds or had they been modified). The invoker is "user" by default, but it may be overridden by a consuming tool (e.g. Hive, CI).
```

Comment:
Can we update [this][1] too?

/hold

[1]: https://github.com/openshift/cluster-version-operator/blob/05f3d334cec0f3e187c9a64a55934a1c5011122b/pkg/cvo/metrics.go#L94

Reply:
I'm open to it. What do you think it should say? It's very terse right now, but it _is_ accurate.

Reply:
Seems like we should mention the `openshift-install` and `other` values for `type`.  And point out that `version` is the invoker version, not the cluster version, just to be extra clear?  Maybe wording like:

> Reports info about the installation process and, if applicable, the install tool.  The type is either "openshift-install", indicating that openshift-install was used to install the cluster, or "other", indicating that an unknown process installed the cluster.  The invoker is "user" by default, but it may be overridden by a consuming tool.  FIXME: something about version...

Based on your example, looks like we set an installer version even when `type=other`?  What if some crazy person goes and installs a cluster from scratch, without even using `openshift-install` to create Ignition configs?  Do we want to say something explicit about `version` always being the version of `openshift-install` that was involved in installation, even if the user did something else to actually spin up the initial cluster resources?

Reply:
Oops, this fell off of my radar. I've updated the description. Let me know what you think.